database/seeds/002-weddings.js

```diff
@@ -0,0 +1,23 @@
+
+exports.seed = function(knex) {
+  return knex("weddings").insert([
+    {
+      slug: "Brey&Abby",
+      // date: 2020-09-21,
```

Comment:
Postgres uses the `timestamp` data type when the Knex `.datetime()` function is used. Thus, an example would like below:

```suggestion
      date: "2020-06-22 19:10:25",
```